File: Wiki/C/code/The_C_Programming_Language/01_Introduction/1.02-Variables_and_Arithmetic_Expressions/04.c
// Purpose: Use more conversion specifications in `printf()`.
// Reference: page 13 (paper) / 27 (ebook)

#include <stdio.h>

    int
main(void)
{
    int num = 123;
    char ch = 'A';
    char str[] = "Hello, world!";

    // `%o`
    printf("%o\n", num);
    //     173

    // `%x`
    printf("%x\n", num);
    //     7b

    // `%c`
    printf("%c\n", ch);
    //     A

    // `%s`
    printf("%s\n", str);
    //     Hello, world!

    return 0;
}

File: Wiki/lua/code/Programming_in_Lua_3rd_edition/ch02_Types_and_Values/exercises/04.lua
-- Purpose: How can you embed the following piece of XML as a string in Lua?
--
--     <![CDATA[
--       Hello world
--     ]]>

-- Reference: page 19 (paper) / 38 (ebook)


-- The issue  is that the string  contains `]]` which conflicts  with the ending
-- delimiter of the syntax for raw strings (`[[ ... ]]`).
-- The  solution  is to  include  an  equal sign  between  the  brackets of  the
-- delimiters:

--     v
print([=[
<![CDATA[
  Hello world
]]>]=])
--  ^
